perception_dataset/rosbag2/autoware_msgs.py

```diff
@@ -198,7 +198,7 @@ def get_category_name(classification: List[ObjectClassification]) -> str:
 
 
 def parse_traffic_lights(
-    roi_msg: TrafficLightRoiArray, signal_msg: TrafficSignalArray
+    roi_msg: TrafficLightRoiArray, signal_msg: TrafficLightArray
```

Comment:
```suggestion
    roi_msg: TrafficLightRoiArray, traffic_light_array_msg: TrafficLightArray
```
Would you change the argument name (and the following part) accordingly?

Reply:
Renamed in https://github.com/tier4/tier4_perception_dataset/pull/80/commits/755239168decae8f10de4ca53fe3461ae8a4c9df